docs/_pages/download.md

```diff
@@ -0,0 +1,36 @@
+permalink: /download
+toc: true
+last_modified_at: 2019-12-30T15:59:57-04:00
+---
+
+## Release 0.8.0
+* Source Release : [Apache Hudi 0.8.0 Source Release](https://downloads.apache.org/hudi/0.8.0/hudi-0.8.0.src.tgz) ([asc](https://downloads.apache.org/hudi/0.8.0/hudi-0.8.0.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.8.0/hudi-0.8.0.src.tgz.sha512))
+* Release Note : ([Release Note for Apache Hudi 0.8.0](/releases.html#release-080-docs))
+
+## Release 0.7.0
+* Source Release : [Apache Hudi 0.7.0 Source Release](https://downloads.apache.org/hudi/0.7.0/hudi-0.7.0.src.tgz) ([asc](https://downloads.apache.org/hudi/0.7.0/hudi-0.7.0.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.7.0/hudi-0.7.0.src.tgz.sha512))
+* Release Note : ([Release Note for Apache Hudi 0.7.0](/releases.html#release-070-docs))
+
+## Release 0.6.0
+* Source Release : [Apache Hudi 0.6.0 Source Release](https://downloads.apache.org/hudi/0.6.0/hudi-0.6.0.src.tgz) ([asc](https://downloads.apache.org/hudi/0.6.0/hudi-0.6.0.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.6.0/hudi-0.6.0.src.tgz.sha512))
+* Release Note : ([Release Note for Apache Hudi 0.6.0](/releases.html#release-060-docs))
+
+## Release 0.5.3
+* Source Release : [Apache Hudi 0.5.3 Source Release](https://downloads.apache.org/hudi/0.5.3/hudi-0.5.3.src.tgz) ([asc](https://downloads.apache.org/hudi/0.5.3/hudi-0.5.3.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.5.3/hudi-0.5.3.src.tgz.sha512))
+* Release Note : ([Release Note for Apache Hudi 0.5.3](/releases.html#release-053-docs))
+
+## Release 0.5.2-incubating
+* Source Release : [Apache Hudi 0.5.2-incubating Source Release](https://downloads.apache.org/incubator/hudi/0.5.2-incubating/hudi-0.5.2-incubating.src.tgz) ([asc](https://downloads.apache.org/incubator/hudi/0.5.2-incubating/hudi-0.5.2-incubating.src.tgz.asc), [sha512](https://downloads.apache.org/incubator/hudi/0.5.2-incubating/hudi-0.5.2-incubating.src.tgz.sha512))
+* Release Note : ([Release Note for Apache Hudi 0.5.2](/older-releases#release-052-incubating-docs))
+
+## Release 0.5.1-incubating
+* Source Release : [Apache Hudi 0.5.1-incubating Source Release](https://downloads.apache.org/incubator/hudi/0.5.1-incubating/hudi-0.5.1-incubating.src.tgz) ([asc](https://downloads.apache.org/incubator/hudi/0.5.1-incubating/hudi-0.5.1-incubating.src.tgz.asc), [sha512](https://downloads.apache.org/incubator/hudi/0.5.1-incubating/hudi-0.5.1-incubating.src.tgz.sha512))
+* Release Note : ([Release Note for Apache Hudi 0.5.1](/older-releases#release-051-incubating-docs))
+
+## Release 0.5.0-incubating
+* Source Release : [Apache Hudi 0.5.0-incubating Source Release](https://downloads.apache.org/incubator/hudi/0.5.0-incubating/hudi-0.5.0-incubating.src.tgz) ([asc](https://downloads.apache.org/incubator/hudi/0.5.0-incubating/hudi-0.5.0-incubating.src.tgz.asc), [sha512](https://downloads.apache.org/incubator/hudi/0.5.0-incubating/hudi-0.5.0-incubating.src.tgz.sha512))
```

Comment:
guess this is the right prefix for urls https://downloads.apache.org/hudi/0.5.0-incubating/ for incubating releases. there is no "incubator" before /hudi/